cmd/topicctl/subcmd/get.go

```diff
@@ -225,10 +235,42 @@ func partitionsCmd() *cobra.Command {
 			}
 			defer adminClient.Close()
 
+			status, valid := cli.StringToPartitionStatus(partitionsConfig.status)
```

Comment:
The way I implemented Enum flags in my PRs was with [cobra.Var](https://github.com/segmentio/topicctl/pull/150/files#diff-efc1a82039cd858b5e3a80f6bfaf5c1c587570afcd9a92ad72f95c28150b58deR353). This appeared to be the idiomatic way of doing this with Cobra. We should be consistent so there aren't multiple ways of doing this in topicctl

Reply:
Will look into it